app/src/main/res/layout/view_week_at_a_glance_meal.xml

```diff
@@ -29,4 +29,14 @@
         app:layout_constraintStart_toStartOf="parent"
         tools:text="Ice Cream"/>
 
+    <ImageView
+        android:id="@+id/week_at_a_glance_menu"
+        android:layout_width="wrap_content"
+        android:layout_height="wrap_content"
+        android:text="@string/week_at_a_glance_delete_menu"
+        android:padding="@dimen/week_at_a_glance_meal_padding"
+        android:src="@drawable/ic_more_vert_black_24dp"
```

Comment:
Let's also set `android:background="?selectableItemBackgroundBorderless"` to add the touch ripple here.